libs/portals/shared-modules/delegations/src/components/Modal/Modal.css.ts

```diff
@@ -21,6 +21,8 @@ export const modal = style({
 })
 
 export const modalInner = style({
+  maxHeight: '100vh',
```

Comment:
We should also add the shadow above the footer if there is content to scroll.

Reply:
Done